README.md

```diff
@@ -134,8 +146,8 @@ Already installed for Python language and up to date
     - Parameters: `scan`
   - Name: Update
     - Parameters: `update`
-  - Name: Yeti
-    - Parameters: `yeti`
+  - Name: build
```

Comment:
I caught this when I renamed the "yeti" command to "build", so I suspect there'll be a merge conflict here.